### ISSUE TITLE
Reference for ARM

### ISSUE THREAD

**Issue by mxms0** [bnagy/cgasm#11]
Don't mean to bloat this, but it'd be really neat-o if cgasm also had an ARM reference. While yes, the ISA is small, it's the semantics that I care about. 

https://developer.arm.com/-/media/developer/products/architecture/armv8-a-architecture/A64_v82A_ISA_xml_00bet3.1.tar.gz ;-)

**Comment by AntonioND** [comment]
Yes, it would be really useful to have both the 32 and 64 instruction sets.
